src/spesh/dump.c

```diff
@@ -90,6 +90,7 @@ static void dump_bb(MVMThreadContext *tc, DumpStr *ds, MVMSpeshGraph *g, MVMSpes
     MVMSpeshIns *cur_ins;
     MVMint64     i;
     MVMint32     size = 0;
+    MVMuint32    line_number;
```

Comment:
What warning does this fix? Generally, it's better to declare variables in the narrowest scope they'll be used, and so this change would seem to make things worse.